src/components/member-card/index.js

```diff
@@ -31,8 +27,7 @@ const Card = ({ developerInfo }) => {
       {!isMember && dev && <SuperUserOptions username={username} />}
       <motion.img
         layoutId={username}
-        src={`${img_url}?${Math.random() * 100}`}
-        onError={brokenImageHandler}
+        src={img_url}
         className={
           isMember ? classNames.imgContainer : classNames.imgContainerNewMember
```

Comment:
I think we should handle the case if the Cloudinary image is broken or fails to load, what do you think :thinking:

Reply:
yes, you are right. I have reverted this.